src/_posts/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative.md

```diff
@@ -0,0 +1,351 @@
+
+{% include soft-mod-cheat-sheet-banner.html %}
+
+## Why would you want to build a business case in the first place?
+
+The answer is simple: to get support and focus. While it may be clear to you that there is a problem that needs to be solved, most of the time, you are not working in a vacuum. Many actors in the organisation will have a direct impact on the feasibility and success of the project.
+
+You might be trying to get buy-in from other business areas, the workforce, or financial support required to make it happen. A good business case will allow you to engage the people you are trying to influence as well as provide a sound foundation for the whole project.
+
+Not having a business case would be like going into battle without a plan.
+
+## What is essential to consider when building your business case for a Software Modernisation project?
+
+There are many areas to consider when preparing a business case for a modernisation project. Still, most of the time, you can be quite safe if you focus on these three factors: the audience, what they need to know, and what they pay off is.
+
+### Audience
+
+Knowing your audience is one of the aspects you should keep in mind while creating your proposal. It will dictate most of your communication style, wording, needs, and benefits or risks that you will want to emphasise.
+
+One crucial aspect to consider is that although you may be talking to a broad audience and an ample range of stakeholders, not all of them are equally important. So start by mapping out your audience and prioritising them is a must. It will be impossible to have a productive conversation if you don't know who your audience is.
+
+![Stakeholder priortisation diagram]({{site.baseurl}}/assets/custom/img/blog/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative/stakeholder-priortisation.jpg)
+
+An excellent way to start is to list out who your stakeholders are. Then prioritise them according to two dimensions: the influence they have, and how much impact your initiative will have on them. Once that is done, you can start with stakeholders who are in the high-influence, high-impact quadrant, and move on from there.
+
+### What they need to know
+
+What is the most critical information your audience needs to know?
+
+You will have already done a lot of research and may well have a good idea of how the project will eventually run, but beware of providing too much unnecessary detail.
+
+The main topics you should focus on:
+
+- The need or problem that you are trying to solve
+
+- The high-level approach or plan to get it done
+
+- What the payoff is for the audience and business
+
+
+{% if site.lang == "en" %}On this particular topic, you may want to check out our <a href="https://info.codurance.com/{{ site.lang }}/case-for-change-questionnaire" target="_blank">list of questions to help you gather the information that you need as part of your research</a>. 
```

Comment:
@keith-smale let's change this link as well so it's always to the English version?

Reply:
Good spot didn't see that there was one in the post too. Will get that sorted.